tools.go

```diff
@@ -0,0 +1,8 @@
+// +build tools
```

Comment:
@squat This was the missing piece https://github.com/golang/go/wiki/Modules#how-can-i-track-tool-dependencies-for-a-module